proposals/NNNN-same-app-from-multiple-devices.md

```diff
@@ -0,0 +1,80 @@
+# Support running the same app from multiple devices at the same time
+
+* Proposal: [SDL-NNNN](NNNN-same-app-from-multiple-devices.md)
+* Author: [Zhimin Yang](https://github.com/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+This proposal proposes that SDL shall support running the same app from multiple devices at the same time. Currently, SDL only allows one instance of the same app to register with SDL.  However, there are valid use cases for running the same app from two or more devices at the same time. 
+
+For example, two `Pandora` apps are running on an iPhone and an android phone respectively. Both devices connected to the vehicle’s infotainment system. They are streaming music to front and rear audio respectively.
+
+As another example, two climate-remote-control apps are running on two devices that both connected to the vehicle. One used by the front passenger, the other used by a passenger sit in the rear seat. They control the front and rear climate features separately. 
+
+## Motivation
+
+SDL supports the following transport types, Bluetooth, USB-AOA, and TCP (WiFi) out of the box. OEMs can also extend that list to include iAP over BT or iAP over USB. For details, please see link 
+https://smartdevicelink.com/en/docs/hmi/master/basiccommunication/updatedevicelist/ . It is possible that multiple devices connect to SDL at the same time using either the same (TCP) or different transports. 
+
+It is also possible that multiple devices have the same version of SDL app installed. When connects to SDL from different devices, the apps will have the same information regarding app name, app id assigned by policy, app icon, voice recognition name, etc. For details, please see definition of `HMIApplication` and `RegisterAppInterface`. 
+
+RPC `UpdateDeviceList` defines that the device list array has 0-100 elements. That is big enough for any real world scenarios. The maximum number of devices connected at the same time depends on the hardware and software configurations of the system running SDL. 
```

Comment:
Change `That is big enough for any` to  `This is big enough for any`

Reply:
updated